test/mlir/onnx/onnx_lowering.mlir

```diff
@@ -2995,20 +2995,17 @@ func @test_tile1(%arg0 : tensor<4x8xf32>) -> tensor<*xf32> {
   %0 = "onnx.Constant"() { value = dense<[3, 2]> : tensor<2xi64>} : () -> tensor<2xi64>
   %1 = "onnx.Tile"(%arg0, %0) : (tensor<4x8xf32>, tensor<2xi64>) -> tensor<*xf32>
   return %1 : tensor<*xf32>
-  // CHECK: [[INDEX_MAP:#.+]] = affine_map<(d0)[s0] -> (d0 mod s0)>
+  // CHECK: [[INDEX_MAP0:#.+]] = affine_map<(d0) -> (d0 mod 4)>
```

Comment:
I would recommend to move this test to the onnx_lowering_with_canonicalize.mlir because it will get rid of the maps and move it into the computation. I don't know how safe it is to check things before the CHECK-LABEL. But I am a very naive FileCheck user, so it may be safe. Anyway, I think its better to canonicalize and see the more readable output.